e2e/src/api/specs/person.e2e-spec.ts

```diff
@@ -185,13 +194,13 @@ describe('/people', () => {
         .set('Authorization', `Bearer ${admin.accessToken}`)
         .send({
           name: 'New Person',
-          birthDate: '1990-01-01T05:00:00.000Z',
+          birthDate: '1990-01-01',
```

Comment:
Not sure if dates with a time should be accepted. I've removed the time part for now.